### PR TITLE
Dịch "lenet_vn" - Phần 5

### LINKED ISSUE #1302: Dịch "lenet_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-neural-networks/lenet_vn.md#L214-L291

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by DacDinh147**
Bản dịch thử nghiệm

## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1302

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 1]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
 Since each epoch takes tens of seconds to run, we visualize the training loss in a finer granularity.
 -->
 
-*dịch đoạn phía trên*
+Hàm huấn luyện `train_ch6` thì cũng giống như hàm huấn luyện `train_ch3` đã được định nghĩa tại phần :numref:`sec_softmax_scratch`.
+Bởi vì hiện tại chúng ta phải xử lý một mạng có tới hàng chục tầng, nên hàm này chỉ hỗ trợ các mô hình được xây dựng trên thư viện Gluon.
```
Comment (rootonchair) [inline chapter_convolutional-neural-networks/lenet_vn.md:246]:
```suggestion
Bởi vì hiện tại chúng ta phải xử lý một mạng có tới hàng chục tầng, nên hàm này chỉ hỗ trợ các mô hình được xây dựng bằng thư viện Gluon.
```

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 2]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
 Since each epoch takes tens of seconds to run, we visualize the training loss in a finer granularity.
 -->
 
-*dịch đoạn phía trên*
+Hàm huấn luyện `train_ch6` thì cũng giống như hàm huấn luyện `train_ch3` đã được định nghĩa tại phần :numref:`sec_softmax_scratch`.
```
Comment (minhduc0711) [inline chapter_convolutional-neural-networks/lenet_vn.md:245]:
```suggestion
Hàm huấn luyện `train_ch6` khá giống với hàm huấn luyện `train_ch3` đã được định nghĩa tại :numref:`sec_softmax_scratch`.
```

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 3]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
...
+Bởi vì hiện tại chúng ta phải xử lý một mạng có tới hàng chục tầng, nên hàm này chỉ hỗ trợ các mô hình được xây dựng bằng thư viện Gluon.
```
Comment (minhduc0711) [inline chapter_convolutional-neural-networks/lenet_vn.md:246]:
```suggestion
Bởi vì hiện tại chúng ta phải làm việc với một mạng có tới hàng chục tầng, hàm này chỉ hỗ trợ các mô hình được xây dựng bằng thư viện Gluon.
```

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 4]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
...
+Chúng ta sẽ khởi tạo bộ tham số của mô hình trên thiết bị đã được chỉ định bởi hàm `ctx`, lần này chúng ta sẽ sử dụng bộ khởi tạo Xavier.
```
Comment (minhduc0711) [inline chapter_convolutional-neural-networks/lenet_vn.md:247]: `ctx` là biến chứ nhỉ
```suggestion
Chúng ta sẽ khởi tạo bộ tham số của mô hình trên thiết bị đã được chỉ định bởi `ctx`, lần này sử dụng bộ khởi tạo Xavier.
```

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 5]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
...
+Hàm mất mát và thuật toán huấn luyện sẽ vẫn dùng hàm entropy chéo làm hàm mất mát và phương pháp hạ gradient ngẫu nhiên cho những batch dữ liệu nhỏ làm thuật toán huấn luyện.
```
Comment (minhduc0711) [inline chapter_convolutional-neural-networks/lenet_vn.md:248]: `minibatch` giữ nguyên nhé
```suggestion
Ta vẫn sử dụng hàm mất mát entropy chéo và thuật toán huấn luyện là phương pháp hạ gradient ngẫu nhiên theo minibatch.
```

File: chapter_convolutional-neural-networks/lenet_vn.md [hunk 6]
```diff
@@ -241,7 +242,11 @@ The loss function and the training algorithm still use the cross-entropy loss fu
...
+Bởi vì mỗi epoch tốn khoảng hàng chục giây để chạy, chúng ta sẽ vẽ biểu đồ đường biểu diễn giá trị mất mát khi huấn luyện để thể hiện quá trình chi tiết hơn.
```
Comment (minhduc0711) [inline chapter_convolutional-neural-networks/lenet_vn.md:249]:
```suggestion
Bởi vì mỗi epoch tốn khoảng hàng chục giây để chạy, chúng ta sẽ vẽ biểu đồ đường biểu diễn giá trị mất mát huấn luyện một cách chi tiết hơn (nhiều giá trị hơn). 
```

Comment (lkhphuc) [reply chapter_convolutional-neural-networks/lenet_vn.md:249]:
```suggestion
Bởi vì mỗi epoch tốn khoảng hàng chục giây để chạy, chúng ta sẽ vẽ đường biểu diễn giá trị mất mát huấn luyện với nhiều giá trị chi tiết hơn. 
```
